config/routes.rb

```diff
@@ -11,6 +11,8 @@
 
   resources :cards
   root 'home#index'
+  get 'check', to: 'cards#check'
+  post 'compare', to: 'cards#compare'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.